test/parallel/test-fs-writev.js

```diff
@@ -0,0 +1,100 @@
+// Flags: --expose_externalize_string
+
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
+const tmpdir = require('../common/tmpdir');
+
+tmpdir.refresh();
+
+const expected = 'ümlaut. Лорем 運務ホソモ指及 आपको करने विकास 紙読決多密所 أضف';
+const getFileName = () =>
+  path.join(tmpdir.path,
+            `writev${Math.random().toString(36).substring(2, 15)}.txt`);
+
+/* eslint-disable no-undef */
+common.allowGlobals(externalizeString, isOneByteString, x);
```

Comment:
None of these globals seem to be used

Reply:
When I delete this I get `AssertionError [ERR_ASSERTION]: Unexpected global(s) found: externalizeString, isOneByteString, x`

Reply:
Did you also remove the `// Flags: ...` comment at the top of the file?